wallet.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import hathorLib from '@hathor/wallet-lib';
+import MemoryStore from './store';
```

Comment:
You can import directly here the `MemoryStore` from wallet-lib

Reply:
Fixed. Thanks!